plugins/caches/delete.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const api = require('./request');
+
+module.exports = () => ({
+    method: 'DELETE',
+    path: '/caches/{scope}/{id}',
+    config: {
+        description: 'API to delete cache using scope and id',
+        notes: 'Deletes the cache for pipeline, job or event using its id',
+        tags: ['api', 'cache'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'pipeline', '!guest']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: async (request, reply) => {
+            const res = await api.invoke(request);
```

Comment:
if the request fails, even after retries, will this handler handle that error and pass error message back to the UI to display?

Reply:
Pass, the message back to the UI. Also, I think the UI should show whether cache was cleared or not. Currently it doesn't do anything